x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/create_connector.tsx

```diff
@@ -137,6 +147,33 @@ export const CreateConnector: React.FC = () => {
     ),
   };
 
+  useUnsavedChangesPrompt({
+    cancelButtonText: i18n.translate(
+      'xpack.enterpriseSearch.createConnector.unsavedPrompt.cancel',
+      {
+        defaultMessage: 'Keep creating',
```

Comment:
```suggestion
        defaultMessage: 'Continue setup',
```

Reply:
Asking @leemthompo for his thoughts on this content changes

Reply:
LGTM